src/index.ts

```diff
@@ -181,11 +181,6 @@ async function parseTarget(distro: string | undefined, platform: string, archs:
       { value: 'darwin', priority: 1 },
       { value: 'macos', priority: 1 }
     ];
-  } else if (['rhel80', 'rhel8'].includes(platform)) {
-    return [
-      { value: 'rhel80', priority: 1 },
-      { value: 'rhel8', priority: 2 }
-    ];
```

Comment:
@kmruiz Was this being used somewhere? `platform` is generally supposed to be `linux` for all the Linuxes

Reply:
It was used for the CLI if IIRC. I think it was this command:

`npx mongodb-download-url --version 6.0.x --platform rhel8`

That was failing.

Reply:
Yeah – was it expected to work? `--distro rhel8` should work fine 🙂